controllers/tmf-apis/usageManagement.js

```diff
@@ -105,8 +113,8 @@ var usageManagement = (function() {
     //////////////////////////////////////////////////////////////////////////////////////////////
 
     var validators = {
-        GET: [utils.validateLoggedIn, tmfUtils.filterRelatedPartyFields, checkFilters],
-        POST: [validateApiKey],
+        GET: [utils.validateLoggedIn, tmfUtils.filterRelatedPartyFields, checkFilters, checkProductId],
+        POST: [],
```

Comment:
You should add a method not allowed validator for now in the POST, otherwise this API would be open